backend/tests/unit/api/v0/tools/test_tools.py

```
@@ -0,0 +1,405 @@
+    service.delete_tool = MagicMock()
+    service.validate_tool = MagicMock()
+    service.get_mock_response = MagicMock()
+    return service
+
+
+@pytest.fixture
+def mock_user_session():
+    """Mock user session for testing."""
+    session = Mock(spec=UserSessions)
+    session.user_id = "test-user"
+    session.oauth_token = "test-oauth-token"
```

Ruff (0.14.1)

Possible hardcoded password assigned to: "oauth_token"

(S105)